draft-ietf-rtcweb-jsep.xml

```diff
@@ -1777,6 +1804,31 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           with these exceptions: <list style="symbols">
               <t>The "s=" and "t=" lines MUST stay the same.</t>
 
+              <t>If any RtpTransceiver has been added, and there
+              exists an m= section with a zero port in the current
+              local description or the current remote description,
+              that m= section MUST be recycled by generating an m=
+              section for the added RtpTransceiver as if the m=
+              section were being added to the session description, placed at
+              the same index as the m= section with a zero port.</t>
+
+              <t>If an RtpTransceiver is stopped and is not associated
```

Comment:
If the RtpTransceiver is not associated with an m=section, that means the the m= section has been recycled, which means there is a new RtpTransceiver taking it's place, and the m=section will be generated.  In other words, it's impossible to end up needing to generate an m=section per SDP rules without having an RtpTransceiver backing it.